cli/tools/bundle/mod.rs

```diff
@@ -979,13 +1077,76 @@ impl DenoPluginHandler {
         if media_type == deno_media_type::MediaType::Unknown {
           return Ok(None);
         }
-        (specifier, media_type, media_type_to_loader(media_type))
+        (specifier, media_type)
       };
+
     let graph = self.module_graph_container.graph();
     let module_or_asset = self
       .module_loader
       .load(&graph, &specifier, None, requested_type)
-      .await?;
+      .await;
+    let module_or_asset = match module_or_asset {
+      Ok(module_or_asset) => module_or_asset,
+      Err(e) => match e.as_kind() {
+        LoadCodeSourceErrorKind::LoadUnpreparedModule(_) => {
+          let file = self
+            .file_fetcher
+            .fetch_bypass_permissions(&specifier)
+            .await?;
```

Comment:
Can you describe (and preferably put a comment here) when and why this would happen that you need to fetch with permission bypass?

Reply:
Hmm yeah I guess this should use the root permissions.

This case is when you need to load something that isn't in the module graph we construct in the beginning.

Reply:
Changed it to use the permissions container